syntax = "proto3";

service FileTransferService {
  rpc upload(stream FileChunk) returns (UploadStatus) {}
}

message FileChunk { bytes content = 1; }

message UploadStatus { int32 length = 1; }
